server/validateReviews.js

```diff
@@ -0,0 +1,23 @@
+import { body,validationResult } from "express-validator";
+export default function validateReviews(req,res){
+  //const body = req.body
+  body('author').isLength({min: 3}),
+  body('comment').isLength({min: 3}),
+  body('rating').isInt({min:0, max:5}),
```

Comment:
I had to research the `express-validator` library a little be certain, but I believe these functions should be used as middleware, meaning they should be used inside a reoute, e.g. like this:

```js
app.post(
  '/some/url',
  body('author').isLength({ min: 3 }),
  (req, res) => {
    // This is where the logic goes, if all went well
  }
);
```

This is achieved by these functions, by returning a request handler functions. However, that's not how they are being used here. Instead, they are invoked out of context and end up just returning functions that are not used for anything.